extensions/shared/upgrade-nudge/store.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { registerStore } from '@wordpress/data';
+
+const actions = {
+	setPlans( plans ) {
+		return {
+			type: 'SET_PLANS',
+			plans,
+		};
+	},
+
+	fetchFromAPI( url ) {
+		return {
+			type: 'FETCH_FROM_API',
+			url,
+		};
+	},
+};
+
+registerStore( 'wordpress-com/plans', {
```

Comment:
Fine to leave `wordpress-com` here but just noting that Publicize uses `jetpack/` prefix — better for consistency?

Reply:
Not sure, TBH. I was using the prefix to indicate the data source -- which is the WordPress.com REST API, not the self-hosted Jetpack site's :man_shrugging:

As in, the store name represents the data source -- if we were fetching other data from WP.com, I'd probably call the stores accordingly, e.g. `wordpress-com/themes` etc.

Reply:
That reasoning makes sense. :thumbsup: Just wasn't clear so might need documenting [in the docs](https://github.com/Automattic/jetpack/tree/master/extensions#good-to-know-when-developing-gutenberg-extensions)?

Reply:
I kinda make this up as I go :joy: There isn't a lot of precedent for `@wordpress/data`-type stores in Gutenpack (there are a few of one-off API fetches outside of stores, though), so it seems kind of a specific information to put into docs. I can try to generalize it so it's not more confusing than helpful (because of its specificity) :sweat_smile:

Reply:
Totally fine to make it up as we go but once we set a pattern for others to follow, it should be documented unless it's obvious by looking at the code.

Reply:
Mmm, I think in this case I'd call it `jetpack/plans` still.

Reply:
I think what made `jetpack/plans` sound wrong to me is that this code is also synced to WP.com, and it'll get plans information solely for WP.com plans there -- so the `jetpack/` prefix seemed kinda misleading for that.

The inverse, however -- having the `wordpress-com/` prefix in Jetpack code -- seemed acceptable, since WP.com is indeed the source of information here.

However, if you're feeling strongly about this, I can change the prefix to `jetpack/` :)

Reply:
@mtias can you please elaborate _why's_ for your opinion, thanks. :-D

Reply:
The plans section in WP.com also has the Jetpack logo, so it's not that odd to me :)